modin/error_message.py

```diff
@@ -21,10 +21,11 @@ class ErrorMessage(object):
     printed_warnings = set()
 
     @classmethod
-    @logger_decorator("MODIN-ERROR", "ErrorMessage.not_implemented", "debug")
     def not_implemented(cls, message=""):
         if message == "":
             message = "This functionality is not yet available in Modin."
+        logger = get_logger()
+        logger.debug("Modin Error: NotImplementedError: {}".format(message))
```

Comment:
Should we keep these caps? Errors are pretty serious and in my opinion they should be at least as visible as, e.g. PANDAS-API.

Reply:
I can add back the caps, and I'll make the errors at the info level and keep the warnings at debug.